keyboards/atreus62/keymaps/194h/keymap.c

```diff
@@ -95,66 +127,23 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
-	OSM_LCTL,      _______,      XXXXXXX,      _______,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      KC_MUTE,      KC_VOLU,      OSM_RCTL
+	OSM_LCTL,      _______,      XXXXXXX,      _______,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      OSM_RALT,     KC_MUTE,      KC_VOLU,      OSM_RCTL
   ),
   [L5] = LAYOUT(
 	XXXXXXX,       TO(L1),       TO(L2),       TO(L3),       TO(L4),       XXXXXXX,                                  XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,
-	XXXXXXX,       XXXXXXX,      XXXXXXX,      XXXXXXX,      RESET,        XXXXXXX,                                  XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      NO_AO,  
-	XXXXXXX,       XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,                                  XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      NO_OE,        NO_AE,  
+	XXXXXXX,       XXXXXXX,      XXXXXXX,      XXXXXXX,      RESET,        XXXXXXX,                                  XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,
+	XXXXXXX,       XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,                                  XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,
 	XXXXXXX,       XXXXXXX,      XXXXXXX,      KC_CAPS,      XXXXXXX,      XXXXXXX,                                  XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,
 	XXXXXXX,       _______,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX,      XXXXXXX
   )
 };
 
-
-// Unicode Linux
-void matrix_init_user(void) {
-  set_unicode_input_mode(UC_LNX);
-};
-
-
-#define TAP_ONCE(code)  \
-  register_code (code); \
-  unregister_code (code)
-
-
-LEADER_EXTERNS();
-
-// Runs constantly in the background, in a loop. Only supports basic keycodes.
-void matrix_scan_user(void) {
-  LEADER_DICTIONARY() {
-    leading = false;
-    leader_end();
-
-    //probably useless in practice, but cool
-    SEQ_ONE_KEY (KC_P) {
-      tap_random_base64();
-      tap_random_base64();
-      tap_random_base64();
-      tap_random_base64();
-      tap_random_base64();
-      tap_random_base64();
-      tap_random_base64();
-      tap_random_base64();
-    }
-
-    SEQ_ONE_KEY (KC_G) {
-      SEND_STRING("grep something !");
-      register_code(KC_RALT);
-      TAP_ONCE(KC_4);
-      unregister_code(KC_RALT);
-    }
-    SEQ_ONE_KEY (KC_T) {
-      SEND_STRING("tail "); 
-      TAP_ONCE(KC_SLSH);
-      SEND_STRING("f !");
-      register_code(KC_RALT);
-      TAP_ONCE(KC_4);
-      unregister_code(KC_RALT);
-    }
-    SEQ_ONE_KEY (KC_S) {
-      SEND_STRING("sudo !!");
-      TAP_ONCE(KC_ENT);
-    }
-  }
-}
+//TODO: Is there productivity gain in the use of unicode?
```

Comment:
Well, not sure if it helps, but we did add a send string function for unicode, that you may want to check out: https://docs.qmk.fm/#/feature_unicode?id=send_unicode_hex_string

Reply:
It amazes me how fast you work and still have the time to help out in this way.
Thank you.
I'll definitely have a look at your suggestions.

Reply:
Well, both are something that I enjoy doing. :) 

That and working from home makes it a lot easier. :)